protostar/commands/test/test_cases.py

```diff
@@ -17,13 +19,19 @@ class TestCaseResult:
     def get_formatted_file_path(self):
         return log_color_provider.colorize("GRAY", str(self.file_path))
 
+    @abstractmethod
+    def log(self, _: bool = False) -> str:
```

Comment:
```suggestion
    def log(self, include_stdout_section: bool = False) -> str:
```

and please use the same argument name in each override.

you are breaking `kwargs` passing here. parameter names are parts of function contract in Python, following snippet would crash here:

```python
result = get_random_test_case_result()
result.log(include_stdout_section=True)
# or
result.log(_=True)
```